templates/openstack/features.html

```diff
@@ -249,7 +249,7 @@ <h2>Storage</h2>
         <li class="p-list__item is-ticked">Object stores and other higher-level storage constructs</li>
         <li class="p-list__item is-ticked">Hierarchical storage configurations</li>
       </ul>
-      <p><a href="/openstack/storage" class="">Learn more about storage with Ubuntu Advantage for Infrastructure ›</a></p>
+      <p><a href="/ceph" class="">Learn more about storage with Ubuntu Advantage for Infrastructure ›</a></p>
```

Comment:
You could remove the empty class if you wanted but not a blocker on merging this.

Reply:
Done